scripts/changeset/update-changelog.mts

```diff
@@ -1,6 +1,7 @@
 import * as core from "@actions/core";
 import * as github from "@actions/github";
 import { getInfo } from "@changesets/get-github-info";
+import { Octokit } from "@octokit/rest";
```

Comment:
Not sure why this was skipped at first. 🤔

Reply:
https://github.com/FuelLabs/fuels-ts/commit/3c9a1a008b6a723f2be50e01eeac80aa8a4c116d